toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The Good Docs Toolchain
+---
+
+This document summarizes the personae for The Good Docs Toolchain, a collection of tools designed to help users create better technical documentation.
+
+The following are the primary types of users (i.e. personae) targeted by this toolchain. Each has a number of distinct needs, as outlined in the below sections.
+
+## Tech writer
+
+The **Tech Writer** is an individual whose primary role is authoring technical documentation. These may be volunteers in open source projects or employees in commercial organizations. They will typically have a certain level of technical proficiency, although this will range from a simple understanding of technical concepts to full proficiency with tools such as the command line.
+
+As a tech writer...
+
+- I want to use a simple tool to manage docs-as-code so I can concentrate on writing.
+- I want to have access to spell/grammar check so I can take advantage of existing tools.
+- I want to have the option to use the editor of my choice so I don't have to learn something new.
+
+## Developer
+
+Often times **Developers** are called on to document the code or application they've created. In business this may be a matter of convenience, where in open source projects there may simply not be anyone else available. But in either case this is a secondary task added to their main roles, often without enough thought given to the effort required to produce something of quality. These individuals are (obviously) very technically savvy, although this doesn't preclude them from wanting to use GUI tools.
+
+As a developer...
+
+- I want help in deciding what form the required documentation should take so I don't need to learn technical writing theory.
```

Comment:
This is good and to build here -- there are lots of types of documentation and tooling that developers are unaware of...  I'm a huge fan of [ADRS: Architecture Decisions Records](https://github.com/joelparkerhenderson/architecture_decision_record#adr-example-templates) which are often over-looked.  There are multiple forms of the `RFC` format that has been adopted by React, Rust-Lang.